### PR TITLE
Refatorando carrinho

### PR DESCRIPTION
**Pull request by lucasrochabz**

### CODE REVIEW

**Review by leduardopacheco — APPROVED** [review]
🫨

**Review by HiFelipe — APPROVED** [review]
Brabo

**Review by dhitzschkyy — APPROVED** [review]
aprovado

File: src/pages/CartPage/CartPage.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  return(
+    <>
+      <div>
+        <Header />
+        <CartCheckout />
+        <MainFooter />
+      </div>
+    </>
+  )
+}
```
Comment (carlosdigital16) [inline src/pages/CartPage/CartPage.jsx]: Add blank line at the end of the file.

**Review by carlosdigital16 — APPROVED** [review]

**Pull request merged**